dm/tests/dmctl_command/run.sh

```diff
@@ -562,6 +589,40 @@ function run_validator_cmd {
 	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
 		"validation status test" \
 		"validator not found for task" 2
+
+	echo "--> validation resolve-error --all"
+	prepare_for_validator_cmd
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"validation show-error test" \
+		"\"id\": \"1\"" 1 \
+		"\"id\": \"2\"" 1 \
+		"\"id\": \"3\"" 1 \
+		"\"id\": \"4\"" 1
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"validation resolve-error test --all"
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"validation show-error test" \
+		"\"id\": " 0
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"validation status test" \
+		"new\/ignored\/resolved: 0\/0\/4" 1
+
+	echo "--> validation ignore-error --all"
+	prepare_for_validator_cmd
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"validation show-error test" \
+		"\"id\": \"1\"" 1 \
+		"\"id\": \"2\"" 1 \
+		"\"id\": \"3\"" 1 \
+		"\"id\": \"4\"" 1
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"validation ignore-error test --all"
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"validation show-error test" \
+		"\"id\": " 0
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"validation status test" \
+		"new\/ignored\/resolved: 0\/4\/0" 1
 }
 
 function run_validator_cmd_error() {
```

Comment:
What about adding some failed cases of inexistent task names or error ids? The former case is supposed to print the `not found` message while the latter one is supposed to succeed quietly.

Reply:
for non-exist error id, we should return a message, not success silently.

add a task in https://github.com/pingcap/tiflow/issues/4426. i t for this case should be added when we fix it.

Reply:
[add invalid task name case](https://github.com/pingcap/tiflow/pull/5633/commits/157554b914af3736a890ea44bfa7067bbc72bb89)